### PR TITLE
Use non dockerhub fedora image

### PR DESCRIPTION
**Pull request by surajssd**

### CODE REVIEW

File: hack/Dockerfile.golang [hunk 1]
```diff
@@ -27,16 +27,16 @@ FROM base-${TARGETARCH}
...
-		gcc \
-		glibc-devel \
-		make \
-		pkg-config \
+	g++ \
+	gcc \
+	glibc-devel \
+	make \
+	pkg-config \
 	; \
```
Comment (bpradipt) [inline hack/Dockerfile.golang:40]: intentional formatting changes ?

**Review by bpradipt — APPROVED** [review]
/lgtm

File: hack/Dockerfile.golang (hunk 1 above)
Comment (surajssd) [reply hack/Dockerfile.golang:40]: Yep intentional, it is put in a separate commit :-)

Comment (katexochen) [reply hack/Dockerfile.golang:40]: What formatter are you using for this? I prefer the previous indention, but it is not super important to me, and I know it is pain to fight against automatic formatters. :wink:

Comment (surajssd) [reply hack/Dockerfile.golang:40]: haha, it is the default one that comes with the Dockerfile plugin for vscode.

**Pull request merged**